pandas/_libs/window/indexers.pyi

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+def calculate_variable_window_bounds(
+    num_values: int,     # int64_t
+    window_size: int,    # int64_t
+    min_periods: object,
+    center: object,
```

Comment:
perhaps leave untyped for now unless these can accept any type. (although I accept that these are unused so could in theory accept anything without raising TypeError)

---

from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions

When adding type hints, avoid using the Any type when possible. Reserve the use of Any for when:

the correct type cannot be expressed in the current type system; and
to avoid Union returns (see above).
Note that Any is not the correct type to use if you want to indicate that some function can accept literally anything: in those cases use object instead.

---

so object is the correct type, unless we want to make these consistent with get_window_bounds signature